src/frontend/src/components/public/PIAFormTabs/review/index.tsx

```diff
@@ -72,6 +72,35 @@ const PIAReview = ({ printPreview }: IReviewProps) => {
   const [reviewNote, setReviewNote] = useState<string>(
     pia?.review?.mpo?.reviewNote || '',
   );
+
+  const enableConfirmButton = () => {
+    if (pia.hasAddedPiToDataElements === false) {
+      if (reviewNote === '') return true;
+      else return false;
+    } else return false;
+  };
+  const enableMPOReviewViewMode = () => {
+    // if it is a delegate PI, we make sure show view mode of this MPO Review part
+    // if and only if the user input some review note for this pia
+    if (
+      pia.hasAddedPiToDataElements === false &&
+      pia?.review?.mpo?.reviewNote &&
+      pia?.review?.mpo?.reviewNote !== '' &&
```

Comment:
we should check if there are not empty spaces

Reply:
this part i think should be ok as we already check if the input is empty, if empty does not allow confirm button enable, so user can not submit empty value to DB